cc7-reana/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM centos:centos7
+
+# Install all required packages
+RUN yum install -y http://linuxsoft.cern.ch/wlcg/centos7/x86_64/wlcg-repo-1.0.0-1.el7.noarch.rpm && \
+    yum install -y HEP_OSlibs git environment-modules which                                      && \
+    yum clean all                                                                                && \
+    curl -o /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py                                 && \
+    chmod +x /tmp/get-pip.py                                                                     && \
+    /tmp/get-pip.py                                                                              && \
+    rm -f /tmp/get-pip.py                                                                        && \
+    pip install PyYAML requests                                                                  && \
+    mkdir -p /opt/ali-bot                                                                        && \
+    cd /opt/ali-bot                                                                              && \
+    git clone https://github.com/alisw/ali-bot /opt/ali-bot
```

Comment:
So, the "publisher" from ali-bot is designed to be backwards-compatible, always - we should always get the latest version. The configuration file we use for fetching the desired version does not come from the repository, so it is pinned.